gatsby-node.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path')
 const _ = require('lodash')
-const webpackLodashPlugin = require('lodash-webpack-plugin')
+const LodashModuleReplacementPlugin = require('lodash-webpack-plugin')
```

Comment:
This is now a [constructor](https://github.com/lodash/lodash-webpack-plugin#usage)